bundle/language_am_ET.properties

```
@@ -1600,7 +1600,7 @@ angal.stat.weeklyepidemsurveilover5
 angal.stat.weeklyepidemsurveilunder5                                                                   = HMIS 33b - ሳምንታዊ ወረርሽኝ. ክትትል. ከ 5 ዓመት በታች
 angal.stat.year                                                                                        = አመት
 angal.supplier.deleted                                                                                 = ተሰርዟል።
-angal.supplier.deleted.col                                                                             = ተሰርዟል።
+angal.common.deleted.txt                                                                             = ተሰርዟል።
```

```suggestion
angal.common.deleted.txt                                                                               = ተሰርዟል።
```
Not sure why you changed to suffix from` col` to `txt`  as it seems to be used as a column header in this PR

Note: there are similar spacing issues in the other languages too.